### PR TITLE
kubeadm: promote the "kubeadm certs" command to GA

### PR DESCRIPTION
**Pull request by neolit123**
The command resided under "kubeadm alpha certs".
It will be promoted to GA in 1.20 as "kubeadm certs".

The existing command "kubeadm alpha certs" will
remain present for one more release, but it will be hidden
from documentation as it is deprecated.

xref https://github.com/kubernetes/kubeadm/issues/2291

### CODE REVIEW

**Comment by neolit123** [comment]
note: the `generated/*` file changes are needed so that this PR can pass CI, but otherwise the docs release process should generate them.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -133,10 +133,10 @@ option. Your cluster requirements may need a different configuration.
       ...
       You can now join any number of control-plane node by running the following command on each as a root:
           kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866 --control-plane --certificate-key f8902e114ef118304e561c3ecd4d0b543adc226b7a07f675f56564185ffe0c07
-      
+
       Please note that the certificate-key gives access to cluster sensitive data, keep it secret!
       As a safeguard, uploaded-certs will be deleted in two hours; If necessary, you can use kubeadm init phase upload-certs to reload certs afterward.
-      
+
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:139]: whitespace cleanup. hoping that this doesn't break anything.

**Comment by neolit123** [comment]
added missing section for `certs generate-csr`.

**Comment by neolit123** [comment]
rebased on top of https://github.com/kubernetes/website/pull/24557

**Comment by neolit123** [comment]
/cc @tengqm 
hi, this PR renames the generated files for "kubeadm_alpha_certs_*" -> "kubeadm_certs_*", with the assumption that since the "kubeadm alpha certs" command will be deprecated in 1.20 it will be hidden from the generator. LMK if this is PR is valid, as is.

**Comment by tengqm** [comment]
I'd like to suggest we put this on hold until we are near the release week. There could be further changes to kubeadm tool. Such changes will lead to the changes in the *generated* docs as well.

**Comment by neolit123** [comment]
SGTM @tengqm . we have no more planned changes for commands, but merging this late seems fine as long as the docs approvers agree.

**Comment by annajung** [comment]
hey @neolit123 wondering if this is part of an enhancement? could you link the enhancement link here?

**Comment by neolit123** [comment]
hi, @annajung no it does not have a tracking issue in k/enhancements. historically this has not been done for kubeadm command promotions.

**Comment by annajung** [comment]
I see, thanks for the clarification! :D

**Comment by annajung** [comment]
@kubernetes/sig-cluster-lifecycle can someone else from sig cluster lifecycle provide a tech review / lgtm?

**Comment by tengqm** [comment]
@annajung This one doesn't need a tech lgtm. The author is from that sig.

**Review by irvifa — COMMENTED** [review]
/unhold

**Review by irvifa — COMMENTED** [review]
/approve

**Review by irvifa — COMMENTED** [review]
/lgtm

**Pull request merged**